docs/user/filesystem.md

```diff
@@ -195,29 +195,283 @@ The contents of multiple pools/containers will be accessible via this
 single kernel mountpoint.
 
 Below is an example of creating and mounting a POSIX container under
-the /tmp/dfuse mountpoint.
+the /scratch_fs/dfuse mountpoint.
 
 ```bash
-$ mkdir /tmp/dfuse
+$ mkdir /scratch_fs/dfuse
 
-$ dfuse -m /tmp/dfuse --pool tank --cont mycont
+$ dfuse -m /scratch_fs/dfuse --pool tank --cont mycont
 
-$ touch /tmp/dfuse/foo
+$ touch /scratch_fs/dfuse/foo
 
-$ ls -l /tmp/dfuse/
+$ ls -l /scratch_fs/dfuse/
 total 0
--rw-rw-r-- 1 jlombard jlombard 0 Jul 10 20:23 foo
+-rw-rw-r-- 1 samirrav samirrav 0 Sep 23 16:31 foo
 
-$ df -h /tmp/dfuse/
+$ df -h /scratch_fs/dfuse/
 Filesystem      Size  Used Avail Use% Mounted on
-dfuse           9.4G  326K  9.4G   1% /tmp/dfuse
+dfuse           537G  5.1G  532G   1% /scratch_fs/dfuse
+$
 ```
 
 ### Launching via fstab
 
 DFuse can be launched via fstab and the standard mount command, it will parse -o options
 and extract pool=<name>,container=<name> if provided and ignore any other filesystem options given.
 
+There are few use cases described below to explain how systemd or /etc/fstab can be used to mount
+the daos container using dfuse.
+
+#### 1> User can mount the file system using mount.fuse3 command
+
+```bash
+$  dmg pool create --scm-size=8G --nvme-size=64G --label=samirrav_pool -u samirrav@
+Creating DAOS pool with manual per-engine storage allocation: 8.0 GB SCM, 64 GB NVMe (12.50% ratio)
+Pool created with 11.11%,88.89% storage tier ratio
+--------------------------------------------------
+  UUID                 : b43b06fe-4013-4177-911c-6d230b88fe6e
+  Service Ranks        : [1-5]
+  Storage Ranks        : [0-7]
+  Total Size           : 576 GB
+  Storage tier 0 (SCM) : 64 GB (8.0 GB / rank)
+  Storage tier 1 (NVMe): 512 GB (64 GB / rank)
```

Comment:
I think that you can assume that the pool is already created like in the previous sections.